src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function(input) {
+      this.result += input;
+    },
```

Comment:
You can use short version:
```suggestion
    add() {
      this.result += input;
    },
```